pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [tool.black]
-target-version=["py36", "py37", "py38", "py39"]
+target-version=["py37", "py38"]
```

Comment:
I'm using Python 3.9. Any reason to now include py39 here?

Reply:
There is no 3.9 setting as of yet, that's why I removed it, sorry, should've said something when I did it.

Reply:
This is my bad, I had to update black on my end, that's why the 3.9 flag was breaking my code.